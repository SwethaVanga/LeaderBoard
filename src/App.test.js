import React from 'react'
import App from './App'
import { Provider } from 'react-redux'
import store from './redux/root'
import renderer from 'react-test-renderer'

it('render correctly App component', () => {
  const AppComponent = renderer
    .create(
      <Provider store={store}>
        <App />
      </Provider>
    )
    .toJSON()
  expect(AppComponent).toMatchSnapshot()
})
