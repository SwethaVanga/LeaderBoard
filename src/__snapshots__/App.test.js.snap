// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly App component 1`] = `
<div
  className="container"
>
  <div
    className="inner"
  >
    <div
      className="title"
    >
      Cube
      <span>
        19
      </span>
       Leaderboard
    </div>
  </div>
</div>
`;
